x-pack/plugins/ingest_manager/public/applications/ingest_manager/layouts/without_header.tsx

```diff
@@ -9,6 +9,8 @@ import { EuiPage, EuiPageBody, EuiSpacer } from '@elastic/eui';
 
 const Page = styled(EuiPage)`
   background: ${props => props.theme.eui.euiColorEmptyShade};
+  flex: 1;
```

Comment:
I think this is breaking the create data source page 
<img width="1571" alt="Screen Shot 2020-04-17 at 2 00 39 PM" src="https://user-images.githubusercontent.com/1336873/79599647-e0f6e680-80b3-11ea-9026-8bc40f3aa30a.png">